open-metadata-distribution/omag-server-platform/docs/http-client-collections/README.md

```diff
@@ -0,0 +1,50 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the Egeria project. -->
+
+# Sample Jetbrains HTTP Scripts
+
+The Jetbrains HTTP Client is a useful utility for testing REST APIs that is both integrated into the Jetbrains tooling 
+and freely available to be run standalone. Further information can be found at 
+[Jebrains HTTP Client](https://www.jetbrains.com/help/idea/http-client-in-product-code-editor.html).
```

Comment:
Jebrains ->JetBrains

The link shows this client is only available in Ultimate IJ. If it is freely available outside IJ Ultimate I suggest supplying that link. If it is only available via IJ Ultimate - it does not seem like a good alternative to Postman.

Reply:
Thank you David, I will update the description with the link. ([Download HTTP Client CLI: CI-friendly way of executing .http files](https://www.jetbrains.com/ijhttp/download/#section=zip-archive))

Reply:
I would suggest keeping the link text simple as just Download. Or maybe 2 links with `Download HTTP Client  Zip` `Download HTTP Client Docker`.